PlayList/src/com/playlist/Test.java

```diff
@@ -0,0 +1,29 @@
+package com.playlist;
+
+import java.util.Scanner;
+
+public class Test {
+
+	public static void main(String[] args) {
+		
+		AudioPlayer aPlayer = new AudioPlayer();
+		
+		PlayerState stopState = new Stop();
+		PlayerState playState = new Play();
+		PlayerState pauseState = new Pause();
+		
+		
+		
+		stopState.play(aPlayer);
```

Comment:
[High] The AudioPlayer should have the responsability to start/stop or pause itself